packages/cli/uni-builder/package.json

```diff
@@ -73,6 +73,7 @@
     "postcss-media-minmax": "5.0.0",
     "postcss-nesting": "12.1.5",
     "postcss-page-break": "3.0.4",
+    "postcss-load-config": "6.0.1",
```

Comment:
I guess we should use v5 because v6 requires Node 18

![image](https://github.com/user-attachments/assets/7fd61576-d1cd-41f8-be09-27ee43a2c213)